obal/__init__.py

```diff
@@ -22,7 +25,23 @@ def find_playbooks(playbooks_path):
     for playbook in playbooks:
         filename = os.path.basename(playbook)
         action = os.path.splitext(filename)[0]
-        playbooks_actions[action] = playbook
+
+        metadata = {
+            'help': '',
+            'vars': {},
+        }
+        try:
+            with open(os.path.join(playbooks_path, '{}.obal.yaml'.format(action))) as obal_metadata:
```

Comment:
I'd actually call that file `{playbook}.metadata.yml`.

also: cheater for using `.yaml` to trick the `glob` above :P